Bridge/DoctrineStatsBundle/DataCollector/DoctrineStatsCollector.php

```diff
@@ -544,9 +544,12 @@ public function identifiersAsString(array $identifiers)
     protected function parseManagedEntities()
     {
         $return = [];
-        /** @var EntityManagerInterface $entityManager */
-        foreach ($this->doctrine->getEntityManagers() as $entityManager) {
-            foreach ($entityManager->getUnitOfWork()->getIdentityMap() as $class => $entities) {
+        foreach ($this->doctrine->getManagers() as $manager) {
+            if (!$manager instanceof EntityManagerInterface) {
```

Comment:
I'm ok with this code, just replace `!` by `$manager instanceof EntityManagerInterface === false` please :)

Reply:
Sure if you want, I mean `instanceof` is a type operator.. it cannot be anything other than a boolean.. Im happy to go with it though if you want?